ecosystem/typescript/sdk/src/coin_client.ts

```diff
@@ -73,9 +69,12 @@ export class CoinClient {
     // Get the receiver address from the AptosAccount or MaybeHexString.
     const toAddress = getAddressFromAccountOrAddress(to);
 
-    const payload = this.transactionBuilder.buildTransactionPayload(func, typeArgs, [toAddress, amount]);
+    const builder = new TransactionBuilderRemoteABI(this.aptosClient, { sender: from.address(), ...extraArgs });
```

Comment:
Does this cache the Remote ABI for a period of time?

Reply:
yes, for 10 minutes 
https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/transaction_builder/builder.ts#L336
cc @jjleng